src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,19 +88,11 @@ const CrossPodWriteForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleCrossPodUpload} autoComplete="off">
+      <form onSubmit={handleFormSubmission} autoComplete="off">
         <div style={formRowStyle}>
-          <label htmlFor="cross-upload-doc">Upload document to username: </label>
+          <p>Upload document to {person}: </p>
```

Comment:
See comment above about `person` for SelectUserContext.

Reply:
fixed